todoapp/app/src/test/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksViewModelTest.java

```diff
@@ -214,4 +218,21 @@ public void getTasksAddViewVisible() {
         // Then the "Add task" action is visible
         assertThat(mTasksViewModel.getTasksAddViewVisible(), is(true));
     }
+
+    @Test
+    public void updateSnackbar_nullValue() {
+        // Get a reference to the class under test
+
```

Comment:
remove this new line
add more comments
add a new line between the act and assert
(same for the other test)